packages/terra-icon/src/icon/IconLocationPin.jsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
+import React from 'react';
+import IconBase from '../IconBase';
+
+const SvgIcon = (customProps) => {
+  const attributes = Object.assign({}, customProps);
```

Comment:
I am curious, what is the purpose of `Object.assign`? It looks like we are just cloning customProps into another variable. Can we directly spread customProps instead?
```jsx
<IconBase {...customProps}>
```

Reply:
I'm not sure what it is either, but i do know that this file is generated, changing the generator is probably out of scope for this change.

Reply:
I made the change to the generator and will include it in the next version release.